OpenRA.Platforms.Default/OpenRA.Platforms.Default.dll.config

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <configuration>
-	<dllmap dll="freetype6" os="linux" target="libfreetype.so.6" />
+	<dllmap dll="freetype6" os="!windows" target="libfreetype.so.6" />
```

Comment:
`!windows` presumably also includes osx, which we won't want?

Reply:
Last `dllmap` rule wins, so the next line with `os="osx"` will override the above.